tasks/main.yml

```diff
@@ -46,7 +63,7 @@
   command: vprotect-node-configure
 
 - name: SSL - import vProtect Server certificate
-  command: '/opt/vprotect/scripts/node_add_ssl_cert.sh {{ server_fqdn }} 8181'
+  command: '/opt/vprotect/scripts/node_add_ssl_cert.sh {{ server_fqdn }} 8181 {{ server_fqdn }} "changeit" {{ ansible_os_family }}'
```

Comment:
czemu serwer jest podany 2 razy? i ogólnie co tam się dzieje w tym skrypcie?

Reply:
W przypadku, gdy 3 i 4 parametr nie został podany do skryptu (tak jak było wcześniej) to przyjmowały defaultową wartość. Z racji, że potrzebowałem dodać kolejny parametr to musiałem je podać, więc przekazuje wartości, które by się do nich i tak przypisały